tgbotapi.api/src/commonMain/kotlin/dev/inmo/tgbotapi/extensions/api/send/RepliesWithChatsAndMessages.kt

```diff
@@ -2045,3 +2048,43 @@ suspend fun TelegramBot.reply(
         )
     }
 }
+
+suspend fun TelegramBot.reply(
```

Comment:
Same replies must be copied in `Replies.kt` with replacement of `toChatId + toMessageId` by `to: AccessibleMessage`